kapitan/inputs/jinja2_filters.py

```diff
@@ -47,6 +51,19 @@ def load_jinja2_filters(env):
     env.filters['shuffle'] = randomize_list
 
 
+def load_jinja2_filters_from_file(env, filter_paths):
+    for path in filter_paths:
+        try:
+            custom_filter_spec = util.spec_from_file_location(path.split('/')[-1].split('.')[0], path)
```

Comment:
Move `path.split('/')[-1].split('.')[0]` to a separate line and assign to a var to make this more readable for other people.

Reply:
os.path.basename(path).split('.')[0]